pkg/cmd/build/build.go

```diff
@@ -71,5 +74,9 @@ func Run(ctx context.Context, namespace, buildKitHost string, isOktetoCluster bo
 		analytics.TrackBuildTransientError(buildKitHost, success)
 		return err
 	}
+	if strings.Contains(err.Error(), "insufficient_scope: authorization failed") {
+		err = okErrors.UserError{E: fmt.Errorf("Authorization failed."),
+			Hint: "Make sure that you are already logged in the registry and have permissions to push."}
```

Comment:
Same here, if we can extract the registry part, it would be great if we can mention it in the message: 'Login into $registry and verify that you have permissions to push images'.